### ISSUE TITLE
nonexistent chamber temperature in OctoPrint 1.5.0 

### ISSUE THREAD

**Issue by LMS0815** [jneilliii/OctoPrint-PlotlyTempGraph#3]
Thank you for this wonderful plugin. Working well on my system (1.5.0)

I found a minor issue.
My printer has no chamber (properly configured). The graph is not generated, which is the expected behavior.
But the legend of the graph is provided. I would expect not seeing it.

![image](https://user-images.githubusercontent.com/8235214/100906331-5d1a2f00-34c9-11eb-8a4c-3aea74b10241.png)

```
browser.user_agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.66 Safari/537.36
connectivity.connection_check: 8.8.8.8:53
connectivity.connection_ok: true
connectivity.enabled: true
connectivity.online: true
connectivity.resolution_check: octoprint.org
connectivity.resolution_ok: true
env.hardware.cores: 4
env.hardware.freq: 1400
env.hardware.ram: 902402048
env.os.bits: 64
env.os.id: linux
env.os.platform: linux
env.plugins.pi_support.model: Raspberry Pi 3 Model B Plus Rev 1.3
env.plugins.pi_support.octopi_version: 0.18.0
env.plugins.pi_support.throttle_state: 0x0
env.python.pip: 20.3
env.python.version: 3.7.3
env.python.virtualenv: true
octoprint.safe_mode: false
octoprint.version: 1.5.0
printer.firmware: Marlin 2.0.5_DW6.2 TM3D SSMDBU

```

**Comment by jneilliii** [comment]
Yeah, I'm seeing the same thing, as discussed in #1. I haven't yet had the time to dig into why that is happening, but suspect there is something different in the core OctoPrint values I'm reacting to that has that in there initially. Was meaning to ask @foosel about this.

**Comment by LMS0815** [comment]
I assume you only need to look at the printer profile:

![image](https://user-images.githubusercontent.com/8235214/100940631-91f1aa80-34f8-11eb-8e00-3e485851e63a.png)

You just need to evaluate the chamber setting.
Same with the nozzle/tool setting.

**Comment by jneilliii** [comment]
yeah, that would be a patch on my end for the issue but bypasses the point of this plugin, being able to graph data provided by any other plugin (like the fork of enclosure plugin that's being worked on), and it just graphs what it receives from octoprint.

**Comment by LMS0815** [comment]
OK, got your point. It's more cosmetics than an issue anyway.

Never the less, the original OctoPrint graph is not showing the chamber in the legend.
Nor is it showing/plotting a second tool (See other reported issue https://github.com/jneilliii/OctoPrint-PlotlyTempGraph/issues/4).

**Comment by jneilliii** [comment]
Thanks to @Dak0r and the PR above this should be resolved in the latest 0.0.4rc4 release. If you can try the latest rc4 version available by copying/pasting the URL below into plugin manager > get more > ...from URL and clicking install an report back that would be helpful.

```
https://github.com/jneilliii/OctoPrint-PlotlyTempGraph/archive/0.0.4rc4.zip
```

**Comment by jneilliii** [comment]
I have confirmation from a forum member that the combination of your fork and my recent updates that this is working now.
